lib/selftest/src/Obelisk/SelfTest.hs

```diff
@@ -516,6 +527,29 @@ testObRunInDirWithMissingStaticFile executable extraArgs mdir = maskExitSuccess
         name1 = T.stripPrefix "• The file " (T.stripStart line1) >>= T.stripSuffix " was not found in static.out"
         name2 = T.stripPrefix "• In the untyped splice: $(static \"" (T.stripStart line2) >>= T.stripSuffix "\")"
 
+testObRunInDirWithMissingStaticFilePath :: String -> [Text] -> Maybe FilePath -> Sh ()
+testObRunInDirWithMissingStaticFilePath executable extraArgs mdir =
+  testObRunInDirWithMissingStaticFileWorker executable extraArgs mdir act checkStaticError
+  where
+    -- | Rename a static file, so that `ob run` will fail with a specific error
+    act = do
+      run_ "sed" ["-e", "s/\\$(static /\\$(staticFilePath /", "-i", "frontend/src/Frontend.hs"]
+      run_ "mv" ["static/obelisk.jpg", "static/obelisk2.jpg"]
```

Comment:
this both feels like a time bomb, but at least the `mv` existed before, so I feel justified.

Reply:
time bomb? We can use `staticWhich` if that is better